Project/Engine/src/sysFrame/Object/Light/SpotLight/SpotLight.cpp

```
@@ -15,6 +15,10 @@ void RawSpotLight::DefaultSetting() {
     light_.falloffStart = std::cos(MathUtils::F_PI / 4.f);
 }
 
+void RawSpotLight::Set(const SpotLight& sl) {
```

Cppcheck (2.10-2)

[style] The function 'Set' is never used.

(unusedFunction)